namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java

```diff
@@ -63,11 +65,36 @@ public RouteInfoManager() {
         this.filterServerTable = new HashMap<String, List<String>>(256);
     }
 
-    public byte[] getAllClusterInfo() {
-        ClusterInfo clusterInfoSerializeWrapper = new ClusterInfo();
-        clusterInfoSerializeWrapper.setBrokerAddrTable(this.brokerAddrTable);
-        clusterInfoSerializeWrapper.setClusterAddrTable(this.clusterAddrTable);
-        return clusterInfoSerializeWrapper.encode();
+    public byte[] getAllClusterInfo(String cluster) {
```

Comment:
You change the signature of the public class here.
How about having another method -- `public byte[] getAllClusterInfo()`? And you won't need calling `getAllClusterInfo(null)` then too ;)

Reply:
I agree @shroman here